src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -100,17 +110,46 @@ def test_azconfig_credential(self, resource_group, location):
                  checks=[self.check('name', credential_list[0]['name'])])
 
 
+class AppConfigIdentityScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(parameter_name_for_location='location')
+    def test_azconfig_identity(self, resource_group, location):
+
+        config_store_name = self.create_random_name(prefix='IdentityTest', length=24)
+
+        location = 'eastus'
+        sku = 'standard'
+        self.kwargs.update({
+            'config_store_name': config_store_name,
+            'rg_loc': location,
+            'rg': resource_group,
+            'sku': sku
+        })
+
+        _create_config_store(self, self.kwargs)
+
+        self.cmd('appconfig identity assign -n {config_store_name} -g {rg}',
+                 checks=[self.check('type', 'SystemAssigned'),
+                         self.check('userAssignedIdentities', None)])
+
+        self.cmd('appconfig identity show -n {config_store_name} -g {rg}',
+                 checks=[self.check('type', 'SystemAssigned'),
+                         self.check('userAssignedIdentities', None)])
+
+
```

Comment:
Could you also help add a test for removing identity?

Reply:
sure, I am still investigating how can prepare a user-assigned identity in scenario test.

---
In reply to: [364517140](https://github.com/Azure/azure-cli/pull/11795#discussion_r364517140) [](ancestors = 364517140)

Reply:
Is your test ready now?

Reply:
yes,  we prepare a user assigned identity in that resource group for testing.